ccd-definition/AuthorisationCaseField/CareSupervision/cafcass.json

```diff
@@ -429,7 +429,35 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "courtBundleTabHeading",
```

Comment:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json

Reply:
reverted. Seems that we don't need to move AuthorisationCaseField to DEPRECATED.json, and there is no such JSON file for AuthorisationCaseField